frontend/src/metabase/css/core/colors.module.css

```diff
@@ -206,6 +206,10 @@
   color: var(--mb-color-text-secondary);
 }
 
+.textTertiary {
```

Comment:
object detail is still using atomic classes so I need to add all new semantic colors for texts